src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    date={article.date}
+    text={article.text}
+  />
```

Comment:
https://mateacademy.slack.com/archives/C01QCEVAKK9/p1621326299132300

```suggestion
  <Article {...article} />
```